landing_page/views.py

```diff
@@ -31,8 +31,11 @@ def profile(request):
 # testing purposes
 def movie(request, movie_id):
     movie = get_object_or_404(Movie, pk=movie_id)
-
-    return render(request, "landing_page/movie.html", {"movie": movie})
+    movie_json = serialize("json", [movie])
+    movie_json = json.loads(movie_json)[0]["fields"]
```

Comment:
we could also prepare the object in django rather than javascript - in fact, for languages, that might need to happen